cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/DefaultPluginConfigurer.java

```diff
@@ -82,6 +84,7 @@ public final <T> T usePlugin(String pluginType, String pluginName, String plugin
       Plugin plugin = addPlugin(pluginType, pluginName, pluginId, properties, selector);
       return pluginInstantiator.newInstance(plugin);
     } catch (PluginNotExistsException | IOException e) {
+      LOG.error("Plugin cannot be added or instantiated. Please refer to stacktrace for cause", e);
```

Comment:
nit - log the plugin type and plugin name also. Also remove `Please refer to stacktrace for cause`